ktor-utils/jvm/src/io/ktor/util/EncodersJvm.kt

```diff
@@ -13,6 +13,17 @@ import java.util.zip.*
 
 private const val GZIP_HEADER_SIZE: Int = 10
 
+// GZIP header flags bits
+private object FLG {
```

Comment:
GzipHeaderFlags ?

Reply:
The `FLG` name was taken from the RFC spec, but I agree that `GzipHeaderFlags` is better. Renamed in commit e209b0b90101794b3505731012c5c0cb1d5d26b1.